mkdocs.yml

```diff
@@ -34,6 +34,8 @@ pages:
   - 'Interactions':
     - 'Overview': interactions/index.md
     - 'Getting Started': interactions/quick-start.md
+    - 'Mobile Messaging':
```

Comment:
I think the Mobile Messaging part should be after the web messaging, no?

Reply:
no strong opinion on this, I only chose this order because messaging sources were originally mobile only

changed in 66d964d